.github/workflows/ruby.yml

```diff
@@ -0,0 +1,41 @@
+name: Ruby
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby-version: [2.3.8, 2.4.6, 2.5.4, 2.6.2, 2.7.2, '3.0.0']
+        gem-version: [3.2.3]
```

Comment:
It looks that we cannot use gem 2.7 on Ruby 2.6/2.7/3.0 and gem 3.x has been bundled with Ruby 2.6 and later.

RubyGems 3.2 tests with Ruby 2.5+:
https://github.com/rubygems/rubygems/blob/faa8cf6cccd2ae18eba7191618ff639cc0ddfd7c/.github/workflows/ubuntu-rubygems.yml

RubyGems 2.7 tests with Ruby 1.8-2.5:
https://github.com/rubygems/rubygems/blob/c66a90aee091bbdf92188c9fdc3439eb2e1684d1/.travis.yml

cf. https://stdgems.org/rubygems/

Reply:
I didn't get that, tests on TravisCI have been doing what?

Reply:
Oh I see. Yeah, I think it adds little benefit to test those combinations. The reason they were still working is that the feature to disallow downgrades is fairly new ([RubyGems 3.2.15](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3215--2021-03-19)), so the downgrades were still working on those older RubyGems. But I agree to remove these entries!